exchanges/okx/okx_websocket.go

```diff
@@ -1518,6 +1518,35 @@ func (ok *Okx) wsProcessBlockPublicTrades(data []byte) error {
 	return trade.AddTradesToBuffer(trades...)
 }
 
+func (ok *Okx) wsProcessBalanceAndPosition(data []byte) error {
```

Comment:
Please add test

Reply:
`TestPushData` includes the push test for the `balance_and_position` channel.